### DISCUSSION TITLE
Any ideas how to use useFieldArray with useTransition (react-spring)?

### DISCUSSION THREAD

**Discussion by gormonn** [react-hook-form/react-hook-form#7659] [Q&A]
Any ideas how to use useFieldArray with [useTransition](https://react-spring.io/hooks/use-transition) (react-spring)?

I tried just pass fields to useTransition hook, but it's seems to get conflict with keys.
For example, we can't correctly remove field from middle of an array.

**Comment by Moshyfawn** [comment]
I'd say there're gonna be issues integrating RHF with transition libs like [react-transition-group](https://github.com/reactjs/react-transition-group) and [react-spring](https://github.com/pmndrs/react-spring) until it's taken into the scope of RHF. See #6075
